tests/cross/src/main/scala/tests/BasePCSuite.scala

```diff
@@ -238,6 +247,9 @@ abstract class BasePCSuite extends BaseSuite {
     }
   }
 
+  object IgnoreScala3Version
```

Comment:
```suggestion
  object IgnoreScala3
```
since it ignores it all?